.theme/app/app.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <NuxtLayout name="default">
+    <NuxtPage />
+  </NuxtLayout>
+</template>
+<script setup>
+import { joinURL } from "ufo";
+
+const config = useRuntimeConfig();
```

Comment:
we mostly use runtime config like below, let's not create inconsistency and refactor to below usage;

```js
const { public: { baseUrl } } = useRuntimeConfig();
```